app/code/Magento/Store/Model/App/Emulation.php

```diff
@@ -111,7 +111,7 @@ public function __construct(
     /**
      * Start environment emulation of the specified store
      *
-     * Function returns information about initial store environment and emulates environment of another store
+     * Function emulates environment of another store
```

Comment:
Thanks for spotting this!
Can you merge this line with the previous one to create a simpler description? Right now we'd be left with two lines that say a very similar thing.

Reply:
@bgorski good shout, that should be better